City-of-Helsinki/drupal-helfi-etusivu
# UHF-5666 Remove title field from target links, replace with built-in …

# For you -paragraph (https://helsinkisolutionoffice.atlassian.net/browse/UHF-5666)

## What was done
* Remove dedicated field for link title, replace with built-in link text in link field

## How to install
* Checkout branch
* Run `make drush-cr drush-cim`

## How to test
* Add a `Target group links` -paragraph to a landing page
* View the page, check that the link text is printed correctly 

## Designers review

One of the checkboxes below needs to be checked (`[x]`)

* [ ] This PR has been visually reviewed by a designer
